.circleci/config.yml

```diff
@@ -88,6 +121,33 @@ jobs:
       - store_artifacts:
           path: junit
 
+  end-to-end-test:
+    <<: *default_container_config
+
+    steps:
+      - *attach_workspace
+
+      - run: sudo sh -c 'echo "127.0.0.1 local.in.ft.com" >> /etc/hosts'
+      - run:
+          name: Prepare neo4j database
+          command: |
+            java -version
```

Comment:
see if you can put this setup script in a make task shared between the 2 jobs that use it now